data/map.txt

```diff
@@ -26879,7 +26879,7 @@ planet Thunder
 
 planet "Tik Klai"
 	attributes wanderer research
-	landscape land/badlands2
+	landscape land/desert9-sfiera
```

Comment:
I usually try to avoid having images with obviously human architecture on non-human planets.